README.md

```diff
@@ -179,7 +178,7 @@ See [more documentation for Proxy Veneur](https://github.com/stripe/veneur/tree/
 
 ### Static Configuration
 
-For static configuration you need one Veneur, which we'll call the _global_ instance, and one or more other Veneurs, which we'll call _local_ instances. The local instances should have their `forward_address` configured to the global instance's `http_address`. The global instance should have an empty `forward_address` (ie just don't set it). You can then report metrics to any Veneur's `udp_address` as usual.
+For static configuration you need one Veneur, which we'll call the _global_ instance, and one or more other Veneurs, which we'll call _local_ instances. The local instances should have their `forward_address` configured to the global instance's `http_address`. The global instance should have an empty `forward_address` (ie just don't set it). You can then report metrics to any Veneur's configured `statsd_listen_addresses` as usual.
```

Comment:
Change `ie` to `i.e.`. I know that's not your mistake, but can fix while here. 😄